### PR TITLE
Text styling 

### PR DESCRIPTION
**Pull request by Shikanime**
Implementation for sfeir-open-source#22.

### CODE REVIEW

File: src/sketch.typings.ts [hunk 1]
```diff
@@ -15,13 +15,40 @@ declare type SketchMSLayer = SketchMSPage | SketchMSSymbolMaster;
...
   symbolID: string;
 }
+declare interface SketchMSStringAttribute {
+  _class: 'stringAttribute';
+  attributes: {
+    MSAttributedStringFontAttribute: {
+      _class: 'fontDescriptor';
+      attributes: {
+        name: 'Roboto-Regular';
+        size: 80;
```
Comment (manekinekko) [inline src/sketch.typings.ts]: why 80?

File: src/app/editor/viewer/lib/parsers/sketch-style-parser.service.ts [hunk 2]
```diff
@@ -123,19 +85,70 @@ export class SketchStyleParserService {
...
-            }
           }
         }
       }
     }
     return parent;
   }
 
-  parseStyleInformation(obj: any, root: any) {
+  private parseText(obj: any, root: any) {
```
Comment (manekinekko) [inline src/app/editor/viewer/lib/parsers/sketch-style-parser.service.ts]: do we have enough info to properly type obj and root?

File: src/app/editor/viewer/lib/parsers/sketch-style-parser.service.ts [hunk 3]
```diff
@@ -123,19 +85,70 @@ export class SketchStyleParserService {
...
+    this.parseTextColor(obj, root);
+    this.parseParagraphStyle(obj, root);
+    this.parseTextFont(obj, root);
+    this.parseAttributeString(obj, root);
+  }
+
+  private parseTextFont(obj: any, root: any) {
```
Comment (manekinekko) [inline src/app/editor/viewer/lib/parsers/sketch-style-parser.service.ts]: can you add some comments on these new functions?

**Comment by manekinekko** [comment]
can you also make sure the unit tests are still in sync and up to date?

**Comment by Shikanime** [comment]
Everything done.

**Comment by manekinekko** [comment]
☸️ Build auto-deployed at: https://xlayers-9b3a5cc.angular.run

**Pull request merged**